### ISSUE TITLE
Can this be updated to v4.3.9? 

### ISSUE THREAD

**Issue by boza111** [YunoHost-Apps/qbittorrent_ynh#5]
The minimum Qbittorrent version should be  v4.3.9 and not 4.2.5 as it is with this release. 4.3.9 seems to be the recommended version. Can this be updated?

**Comment by vanyasem** [comment]
This integration basically installs `qbittorrent-nox` package directly from `apt`, and creates a couple of yunohost-related config files:
https://github.com/YunoHost-Apps/qbittorrent_ynh/blob/7fea4f4856682c7db9d7fdab3bcc9bf2ac5566f7/manifest.toml#L53C17-L53C32

YunoHost internally runs on Debian **11** `bullseye`, which is stuck at the version 4.2.5: https://packages.debian.org/bullseye/qbittorrent-nox

**Comment by vanyasem** [comment]
A hacky solution to this would be to add a custom apt source with newer versions of qbittorrent: https://software.opensuse.org//download.html?project=home%3Anikoneko%3Atest&package=qbittorrent

**Comment by vanyasem** [comment]
TLDR run the following commands on your server, then restart qbittorrent:
```
echo 'deb http://download.opensuse.org/repositories/home:/nikoneko:/test/Debian_11/ /' | sudo tee /etc/apt/sources.list.d/home:nikoneko:test.list
curl -fsSL https://download.opensuse.org/repositories/home:nikoneko:test/Debian_11/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_nikoneko_test.gpg > /dev/null
sudo apt update
sudo apt upgrade
sudo apt autoremove
```

**Comment by vanyasem** [comment]
Not only this will ensure that you always have the latest version, it also defaults to libtorrent 2, which fixes some nasty IP leakage issues: https://github.com/qbittorrent/qBittorrent/issues/17449

**Comment by TheGithubJoshua** [comment]
couldn't this be included in the install script?

**Comment by vanyasem** [comment]
@TheGithubJoshua of course this could be!
In fact there are packages that already include PPAs in their install scripts such as PufferPanel: https://github.com/YunoHost-Apps/pufferpanel_ynh/blob/master/manifest.toml#L58

You can bump my thread on Yunohost forums, but I doubt that it would make a difference
https://forum.yunohost.org/t/qbittorrent-ppa-considerations/28362

I have since stopped using Yunohost for this exact reason - immaturity, and being constantly out of date, in favor of managing official Docker containers with Ansible

**Comment by TheGithubJoshua** [comment]
I could make a PR when I get time.

**Comment by brimwats** [comment]
@TheGithubJoshua did you ever make a pull request?
